app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -1076,127 +1088,164 @@ fun PostListing(
 
     var viewSource by remember { mutableStateOf(false) }
 
-    when (postViewMode) {
-        PostViewMode.Card ->
-            PostListingCard(
-                postView = postView,
-                admins = admins,
-                moderators = moderators,
-                instantScores = instantScores,
-                onUpvoteClick = {
-                    instantScores = instantScores.update(VoteType.Upvote)
-                    onUpvoteClick(postView)
-                },
-                onDownvoteClick = {
-                    instantScores = instantScores.update(VoteType.Downvote)
-                    onDownvoteClick(postView)
-                },
-                onReplyClick = onReplyClick,
-                onPostClick = onPostClick,
-                onSaveClick = onSaveClick,
-                onCommunityClick = onCommunityClick,
-                onEditPostClick = onEditPostClick,
-                onDeletePostClick = onDeletePostClick,
-                onReportClick = onReportClick,
-                onRemoveClick = onRemoveClick,
-                onBanPersonClick = onBanPersonClick,
-                onBanFromCommunityClick = onBanFromCommunityClick,
-                onLockPostClick = onLockPostClick,
-                onFeaturePostClick = onFeaturePostClick,
-                onPersonClick = onPersonClick,
-                onViewSourceClick = {
-                    viewSource = !viewSource
-                },
-                viewSource = viewSource,
-                showReply = showReply,
-                showCommunityName = showCommunityName,
-                fullBody = fullBody,
-                account = account,
-                expandedImage = true,
-                enableDownVotes = enableDownVotes,
-                showAvatar = showAvatar,
-                useCustomTabs = useCustomTabs,
-                usePrivateTabs = usePrivateTabs,
-                blurNSFW = blurNSFW,
-                showPostLinkPreview = showPostLinkPreview,
-                appState = appState,
-                showIfRead = showIfRead,
-                showScores = showScores,
-                postActionbarMode = postActionbarMode,
-            )
+    val swipeState = rememberSwipeActionState(swipeToActionPreset) {
+        when (it) {
+            SwipeToActionType.Upvote -> {
+                instantScores =
+                    instantScores.update(VoteType.Upvote)
+                onUpvoteClick(postView)
+            }
+            SwipeToActionType.Downvote -> {
+                instantScores =
+                    instantScores.update(VoteType.Downvote)
+                onDownvoteClick(postView)
+            }
+            SwipeToActionType.Reply -> {
+                onReplyClick(postView)
```

Comment:
I am not sure why but the reply to post in postlistings actions isnt' working for me at all. It doesn't seem to do anything

Reply:
The one in postview does work but no in the main feed

Reply:
By some reason empty lambda was passed somewhere to postlisting. Fixed it.